scripts/testnet.sh

```diff
@@ -123,6 +123,7 @@ start_testnode_vm() {
 
 # $1 network name
 # $2 docker image
+# NB: Needs to output the bs multiaddrss at the end for the testnet.
```

Comment:
can you explain?

Reply:
Right, issue #986 started happening because this function was not echoing the bootstrap multiaddrss. Because this function is needed in the testnet script in the form `local bs = $(start_testnode_vm ...)` and then used afterward, if `start_testnode_vm` does not echo the address, `bs` is empty.